ui/objects/templates/semantic-ui/objects/Detail.jinja

```diff
@@ -1,359 +1,308 @@
-                  </td>
-                </tr>
-                {% endif %} {% if metadata.restorationWork.restorationMethods %}
-                <tr class="details__div__info-left__box">
-                  <td class="parag">Metody restaurování</td>
-                  <td class="parag">
-                    {% for value in metadata.restorationWork.restorationMethods %}
-                    {{ value.title }}
-                    {% endfor %}
-                  </td>
-                </tr>
-                {% endif %}
-              </tbody>
-            </table>
+                      </div>
+
+                      {% endfor %}
+                    </td>
+                  </tr>
+                  {% endif %} {% if metadata.restorationWork.examinationMethods
+                  %}
+                  <tr>
+                    <td><b>Metody průzkumu</b></td>
+                    <td>
+                      {% for value in
+                      metadata.restorationWork.examinationMethods %}
+                      {{ value.title }}
+                      {% endfor %}
+                    </td>
+                  </tr>
+                  {% endif %} {% if metadata.restorationWork.restorationMethods
+                  %}
+                  <tr>
+                    <td><b>Metody restaurování</b></td>
+                    <td>
+                      {% for value in
+                      metadata.restorationWork.restorationMethods %}
+                      {{ value.title }}
+                      {% endfor %}
+                    </td>
+                  </tr>
+                  {% endif %}
+                </tbody>
+              </table>
             </div>
             {% if metadata.restorationWork.abstract %}
-            <div class="column details__div__info-right">
-              <div>
-                <p class="details__div__info-right parag">Restaurátorský zásah/zaměr</p>
-              </div>
-    
-              <div class="details__div__info-right__text">
-                <p class="parag">
-                  {{ metadata.restorationWork.abstract }}
-                </p>
-              </div>
+            <div class="column">
+              <b>Restaurátorský zásah/zaměr</b>
+
+              <p>
+                {{ metadata.restorationWork.abstract }}
+              </p>
             </div>
             {% endif %}
-            </div>
-    
-    
-    
-        <div class="vr-huge-horiz"></div>
-    
-        <!-- docs -->
-        <div id="details__div__docs"></div>
-    
-    <script> 
-     
-    
-    const accordionTitles = document.querySelectorAll('.title.parag');
-    const accordionContents = document.querySelectorAll('.content');
-    
-    
+          </div>
+
+          <div class="ui divider"></div>
+
+          <!-- docs -->
+          <div id="details__div__docs"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script>
+
+    const accordionTitles = document.querySelectorAll(".title");
```

Comment:
This breaks on production (no explicit script tags allowed to execute), move to components webpack bundle